app/connectors/FileUploadConnector.scala

```diff
@@ -26,29 +26,22 @@ import scala.concurrent.Future
 
 trait FileUploadConnector extends ServicesConfig {
 
-  val http: HttpPost = WSHttp
-
+  val http: HttpPost
   lazy val serviceUrl = baseUrl("file-upload")
   lazy val serviceUrlSuffix = getString("file-upload-url-suffix")
 
-  def getEnvelope()(implicit hc: HeaderCarrier): Future[Option[String]] = {
-    val fileUploadUri = s"$serviceUrl/$serviceUrlSuffix"
-    val requestBody = Json.parse("""{"callbackUrl": "ourCallbackUrl"}""".stripMargin)
-    http.POST[JsValue, Option[String]](fileUploadUri, requestBody,Seq())(implicitly, rds = responseHandler, hc, MdcLoggingExecutionContext.fromLoggingDetails(hc))
-  }
+  def getEnvelope()(implicit hc: HeaderCarrier): Future[HttpResponse] = {
 
-  private val responseHandler = new HttpReads[Option[String]] {
+    val requestBody = Json.parse("""{"callbackUrl": "http://localhost:9673/relief-at-source/bulk/upload-callback"}""".stripMargin)
 
-    override def read(method: String, url: String, response: HttpResponse): Option[String] = {
-      response.status match {
-        case 201 => response.header("Location").map{ locationHeader =>Some(locationHeader)}.getOrElse(None)
```

Comment:
I think we have to deal with any other failure cases the same so either 201 or default and also add logger.error please